.github/workflows/containers.yml

```diff
@@ -11,8 +11,16 @@ env:
 
 jobs:
   build_and_publish:
-    name: "Build and publish containers for all platforms"
+    name: "Build & push"
```

Comment:
Make shorter on purpose, so that GitHub appended matrix parameters don't get truncated in UI.